docker-compose.yml

```diff
@@ -210,11 +210,11 @@ services:
       - env/receipt.env
     networks:
       - sdx-env
-    logging:
-      driver: syslog
-      options:
-        syslog-address: "udp://127.0.0.1:5514"
-        tag: "sdx-mock-receipt"
+    # logging:
```

Comment:
Probably better to just delete this.

Reply:
I was thinking of that, didn't want to do too much in a single PR, but if you're asking for it then that works for me!